tests/src/test/scala/runtime/actionContainers/Python39Tests.scala

```diff
@@ -1,6 +1,3 @@
-#!/usr/bin/env python
```

Comment:
how did git get so confused about this file - this is not a moved file :)

Reply:
I removed this file it is no more possible to "generate" an invalid main... the compilation build the virtual env fails